packages/livekit-rtc/package.json

```diff
@@ -56,9 +65,9 @@
   },
   "scripts": {
     "prebuild": "node -p \"'export const SDK_VERSION = ' + JSON.stringify(require('./package.json').version) + ';'\" > src/version.ts",
-    "build:tsc": "pnpm prebuild && tsc && cp -r src/napi dist/",
-    "build": "pnpm build:tsc && napi build --platform --release --dts native.d.ts --js native.cjs --pipe \"prettier -w\" src/napi",
-    "artifacts": "pnpm build:tsc && napi artifacts",
+    "build:ts": "pnpm prebuild && tsup --onSuccess \"tsc --declaration --emitDeclarationOnly\" && cp -r src/napi dist/ && cp -r src/napi/* dist/",
+    "build": "pnpm build:ts && napi build --platform --release --dts native.d.ts --js native.cjs --pipe \"prettier -w\" src/napi",
```

Comment:
this looks like napi is only building for cjs, is that the behaviour we want?

Reply:
we have a wrapper for esm, napi only generates  CJS

Reply:
update: this doesn't actually work, tried using agents and it broke. unsure where to go from here because napi doesn't do ESM

Reply:
what's the error you're running into?

Reply:
something about not being able to find the `native.cjs` file when running it under cjs. can't reproduce right now, i'm having a different issue related to tsup on Agents in specific

Reply:
upon further testing with node-sdks examples directly this doesn't appear to be reproduceable 🤷‍♀️